sources/packages/api/src/database/entities/student-scholastic-standing.model.ts

```diff
@@ -115,4 +121,17 @@ export class StudentScholasticStanding extends RecordDataModel {
     referencedColumnName: ColumnNames.ID,
   })
   note?: Note;
+  /**
+   * Student assessment associated with this student scholastic standing.
+   */
+  @OneToOne(
+    () => StudentAssessment,
+    (studentAssessment) => studentAssessment.studentScholasticStanding,
+    {
+      eager: false,
+      cascade: ["insert", "update"],
```

Comment:
IMO, can we have cascade:true, instead of ["insert", "update"] as the other options in cascade like the "remove" | "soft-remove" | "recover" are all needed, if these 2 options are only needed.

Reply:
do we need to delete cascasde? i basically used the same logic in student appeals

Reply:
no no, cascade: true

Reply:
![image](https://user-images.githubusercontent.com/77353155/164054341-0fb08ea5-847a-4347-83d8-e331cbe0b91e.png)

Reply:
@andrewsignori-aot @dheepak-aot @henryoforeh-aot Please share your comments on the above, I feel its better to have cascade:true instead of having the properties in the array if its insert and update, i see its already in the appeal model too.

Reply:
Sorry, the comments are not clear to me and maybe I am missing something.
We do not have any requirements to "remove" | "soft-remove" | "recover" and doing any of those actions will be a pretty big deal on reassessments.
IMO, restricting the cascading options to only ["insert", "update"] is the safest way. I did not understand why the `cascade:true` would be better in this scenario. @guru-aot, any thoughts?